src/notifications/components/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import localStyles from './Notification.css'
+
+const Button = props => (
```

Comment:
`<Button>` is a too generic name for something that is solely used for notifs that open a new URL (`NotificationContainer.renderButtons` method). Maybe `<OpenLinkButton>` or something? 😕